src/components/test_main.cc

```diff
@@ -3,6 +3,7 @@
 #include "utils/custom_string.h"
 int main(int argc, char** argv) {
   namespace custom_str = utils::custom_string;
+  INIT_LOGGER("log4cxx.properties", false);

```

Comment:
@LuxoftAKutsan do we need logger in tests?

Reply:
@LuxoftAKutsan It would be great if we can use logger in tests

Reply:
@AKalinich-Luxoft yep, that's would be good to have logging in UT